imap_processing/swapi/swapi_metadata.json

```diff
@@ -0,0 +1,246 @@
+{
+    "global_attrs": {
+        "Project": "STP>Solar-Terrestrial Physics",
+        "Source_name": "IMAP>Interstellar Mapping and Acceleration Probe",
+        "Discipline": "Solar Physics>Heliospheric Physics",
+        "Mission_group": "IMAP>Interstellar Mapping and Acceleration Probe",
+        "PI_name": "Dr. David J. McComas",
+        "PI_affiliation": "Princeton Plasma Physics Laboratory",
+        "File_naming_convention": "source_descriptor_datatype_yyyyMMdd_vNNN",
+        "Data_version": 1,
+        "Descriptor": "swapi>The Solar Wind and Pickup Ion",
+        "TEXT": "The Solar Wind and Pickup Ion (SWAPI) instrument measures several different elements of the solar wind, including hydrogen (H) and helium (He) ions, and, on occasion, heavy ions produced by large events from the Sun. See https://imap.princeton.edu/instruments/swapi for more details. SWAPI level-1 data contains primary, secondary, coincidence counts per ESA voltage step and time. Level-2 data contains the same data as level-1 but counts are converted to rates by dividing counts by time.",
+        "Instrument_type": "Particles (space)"
+    },
+    "imap_swapi_l1_sci": {
+        "global_attrs": {
+            "Logical_file_id": "<to-be-replaced-in-code>",
+            "Data_type": "L1A_SCI-1MIN>Level-1A Science data in 1 minute resolution",
+            "Logical_source": "imap_swapi_l1_sci-1min",
+            "Logical_source_description": "SWAPI Instrument Level-1 Science Data in 1 minute resolution.",
+            "sweep_table": "Sweep table number",
+            "plan_id": "Plan id number",
+            "Project": "STP>Solar-Terrestrial Physics",
+            "Source_name": "IMAP>Interstellar Mapping and Acceleration Probe",
+            "Discipline": "Solar Physics>Heliospheric Physics",
+            "Mission_group": "IMAP>Interstellar Mapping and Acceleration Probe",
+            "PI_name": "Dr. David J. McComas",
+            "PI_affiliation": "Princeton Plasma Physics Laboratory",
+            "File_naming_convention": "source_descriptor_datatype_yyyyMMdd_vNNN",
+            "Data_version": 1,
+            "Descriptor": "swapi>The Solar Wind and Pickup Ion",
+            "TEXT": "The Solar Wind and Pickup Ion (SWAPI) instrument measures several different elements of the solar wind, including hydrogen (H) and helium (He) ions, and, on occasion, heavy ions produced by large events from the Sun. See https://imap.princeton.edu/instruments/swapi for more details. SWAPI level-1 data contains primary, secondary, coincidence counts per ESA voltage step and time. Level-2 data contains the same data as level-1 but counts are converted to rates by dividing counts by time.",
+            "Instrument_type": "Particles (space)"
+        },
+        "epoch": {
```

Comment:
How can we make sure that `epoch` has consistent attributes across all data products?

Reply:
I think this is a great question, and I'm not sure you can with this approach. You'd either have to check in the spreadsheets (maybe link/reference more cells), or check further downstream with I hope what Maxine's approach will provide.

Reply:
Maxine's approach does provide this! And also addresses my concern about global attribute consistency across instruments and products.